obelisk_ws/src/obelisk_ros/launch/obelisk_bringup.launch.py

```diff
@@ -116,6 +117,17 @@ def obelisk_setup(context: launch.LaunchContext, launch_args: Dict) -> List:
             )
         )  # once the node is configured, it will be activated automatically
         obelisk_launch_actions += [configure_event, activate_upon_configure_handler]
+    elif auto_start == "configure":
+        # Just configure all nodes
+        configure_event = EmitEvent(
+            event=ChangeState(
+                lifecycle_node_matcher=launch.events.matches_action(global_state_node),
+                transition_id=lifecycle_msgs.msg.Transition.TRANSITION_CONFIGURE,
+            )
+        )
+        obelisk_launch_actions += [configure_event]
+
+    # if auto_start is false, then no configuration at all
```

Comment:
i would move this to combine with a larger comment at the top of the autostart if statements